Mathlib/GroupTheory/Perm/Finite.lean

```diff
@@ -251,3 +251,5 @@ theorem support_pow_coprime {σ : Perm α} {n : ℕ} (h : Nat.Coprime n (orderOf
 end Fintype
 
 end Equiv.Perm
+
```

Comment:
You seem to still have some extra newlines here.

Reply:
oops…

Reply:
Still

Reply:
sorry, had forgotten to push.